README.md

```diff
@@ -1,14 +1,246 @@
-# Wallet Adapter
+# <center>Solana Wallet Adapter </center>
+![Wallets](wallets.png)
+
+---
+<!--PROJECT SHIELDS -->
+<center>
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+
+</center>
```

Comment:
Let's leave these out for now, none of our other readmes use these shields.io shields AFAIK

Reply:
Thanks for the review and all feedback is duly noted